packages/terra-site/webpack.config.js

```diff
@@ -109,4 +109,7 @@ module.exports = {
       errors: true,
     },
   },
+  resolveLoader: {
```

Comment:
I tried that initially but it didn't work. It sounds like the resolve loader is used to locate webpack loaders where resolve is used to locate package dependencies.